setup

```diff
@@ -17,7 +17,7 @@ if test "$(uname)" = "Darwin"; then
     brew upgrade ansible
   fi
 elif test "$(uname)" = "Linux"; then
-  if test -f /etc/lsb-release -a ! "$(which ansible)"; then
+  if test -f /etc/lsb-release && -a ! "$(which ansible)"; then
```

Comment:
Actually I don't know why when I'm running on Linux the argument `-a` isn't recognize.

```shell
./setup.sh: line 20: test: argument expected
Not supported yet.

GNU bash, version 4.3.46(1)-release (x86_64-pc-linux-gnu)
```

Only runs when I add the AND.

P.S: Sorry about the PR, the flag `-a` shouldnt stay.